lms/templates/problem.html

```diff
@@ -66,28 +66,28 @@ <h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="
         ${_("You have used {num_used} of {num_total} attempts").format(num_used=attempts_used, num_total=attempts_allowed)}
     % endif
     </div>
-
+  </div>
     % if answer_notification_type:
```

Comment:
Need to have @sstack22 / @chris-mike weigh in on the order of these notifications.